components/ui/TooltipComponent.tsx

```diff
@@ -27,7 +29,10 @@ const TooltipComponent: React.FC<TooltipComponentProps> = ({
             align="center"
             style={{ whiteSpace: "normal", width: "200px", lineHeight: "1.5" }}
           >
-            {message}
+            <div className="flex flex-row items-center">
+              {message}
+              <img src={icon} className="w-10 h-10 rounded-full p-2" />
```

Comment:
should we use conditional rendering to only render the img element when the icon prop exists?

Reply:
did icon &&.  thank you!